### DISCUSSION TITLE
Mobile authenticator app not connecting to custom server

### DISCUSSION THREAD

**Discussion by nabeelmoeen** [ente-io/ente#3655] [Q&A]
Hi,
I have setup a self hosted ente auth server behind a caddy reverse proxy.
I am able to ping it and receive the expected Pong response.
But when I use the URL in the custom server on the mobile app it returns an error stating invalid api or no api found.

**Comment by nabeelmoeen** [comment]
After multiple attempts on trying to connect it finally does.
But every new client has the same issue of throwing an invalid endpoint error before randomly connecting on subsequent attempts

**Comment by femme-theory23** [comment]
Im having this issue too, after pulling the latest image. It was working fine previously, and then I tried syncing a new code and nothing was syncing. Cleared out the local database on my PC and logged back in, except this time the endpoint URL is invalid. Help would be appreciated.

**Comment by mngshm** [comment]
@nabeelmoeen can you show the full error message ?

**Comment by tpgalan** [comment]
@nabeelmoeen did you manage to solve it properly? It happens the same to me, but I'm not able to solve it in iOS. It works perfectly on macOS or Windows app, but not in iOS. Does someone has any thoughts?

It might be something with the URL syntax or is this just a matter of the connection?

**Reply by nabeelmoeen** [reply]
Hi, no. Unfortunately it didn't work and I went a diff direction
